srcpkgs/asahi-base/INSTALL

```diff
@@ -1,3 +1,3 @@
-if [ "${ACTION}" = "post" ] && ! [ -d /var/service/speakersafetyd ]; then
-	ln -s /etc/sv/speakersafetyd /var/service
+if [ "${ACTION}" = "post" ] && ! [ -d /etc/runit/runsvdir/default/speakersafetyd ]; then
+	ln -s /etc/sv/speakersafetyd /etc/runit/runsvdir/default
```

Comment:
```suggestion
if [ "${ACTION}" = "post" ] && ! [ -d etc/runit/runsvdir/default/speakersafetyd ]; then
	ln -s /etc/sv/speakersafetyd etc/runit/runsvdir/default
```
this allows it to run in `xbps-install -r /mnt` correctly iirc

Reply:
done.